main.go

```diff
@@ -179,8 +179,9 @@ type Bridge struct {
 	usersByMXID         map[id.UserID]*User
 	usersByUsername     map[string]*User
 	usersLock           sync.Mutex
-	managementRooms     map[id.RoomID]*User
+	spaceRooms          map[id.RoomID]*User
 	managementRoomsLock sync.Mutex
+	managementRooms     map[id.RoomID]*User
```

Comment:
Move this back above `managementRoomsLock`

Reply:
It was fixed on my last commit but it was needed to add spaceRoomsLock